model.py

```diff
@@ -31,7 +31,9 @@
     create_engine,
 )
 from sqlalchemy import exc as sa_exc
-from sqlalchemy import func
+from sqlalchemy import (
+    func,
+)
```

Comment:
This is an odd one.

Reply:
Good catch, I totally missed that when I read this over. Looks like its coming from isort 6.0.0, which only came out a couple weeks ago. There is an issue in their queue that I think tags what is going on: https://github.com/PyCQA/isort/issues/2352

I'll roll back isort to the last version for now, they will probably have it fixed by the time we bump pre-commit deps again.